### ISSUE TITLE
Default Exports from nested files generate invalid types

### ISSUE THREAD

**Issue by elado** [wessberg/rollup-plugin-ts#36]
```ts
	const bundle = await generateRollupBundle([
		{
			entry: true,
			fileName: "index.ts",
			text: `\
          		import X from './bar';
          		export { X }
        	`
		},
		{
			entry: false,
			fileName: "bar.ts",
			text: `\
				interface Foo { n: number; }
				export const fn = (x: Foo): Foo => x;
				export default fn({ n: 0 });
            `
		}
	]);
```

generated types:

```ts
 interface Foo {
    n: number;
}
declare const defaultBarExport: Foo;
declare const X = defaultBarExport;
export { X };
```

Error on `const X = defaultBarExport`

```
A 'const' initializer in an ambient context must be a string or numeric literal or literal enum reference.ts(1254)
```

should probably be:

```ts
declare const X: typeof defaultBarExport;
```

or simply

```ts
declare const X: Foo;
```

**Comment by wessberg** [comment]
Thanks for spotting this, and also thanks for these test cases! Fixed in v1.1.72 which has been released just now.

**Comment by elado** [comment]
awesome, thanks! I'm following the way you fix things (need to get to know TS api better) so I can actually contribute next time.
